lighthouse-core/scripts/i18n/prune-obsolete-lhl-messages.js

```diff
@@ -117,7 +117,7 @@ function getGoldenLocaleArgumentIds(goldenLhl) {
  * (e.g. by picking a new message id).
  */
 function pruneObsoleteLhlMessages() {
-  const goldenLhl = readJson('lighthouse-core/lib/i18n/locales/en-US.json');
+  const goldenLhl = readJson('shared/localization/locales/en-US.json');
```

Comment:
this plugin _almost_ works for us https://www.npmjs.com/package/eslint-plugin-require-path-exists

we could have a lint rule for "readJson(path); path is real", but we'd need a webpack config defining an alias to do it off-the-shelf via this plugin.